src/components/Categories/LoanSpotlight.vue

```diff
@@ -0,0 +1,270 @@
+
+	// no channel that matches the targeted name
+	if (targetedLoanChannel.length === 0) {
+		// return id for fallback channel
+		return fallbackLoanChannel[0]?.id || null;
+	}
+	// targeted channel exists but no loans exist within it
+	if (targetedLoanChannel.length !== 0 && targetedLoanChannel[0].loans.totalCount === 0) {
+		return fallbackLoanChannel[0]?.id || null;
+	}
+	// isolate targeted channel id
+	return targetedLoanChannel[0]?.id || null;
+}
+
+function filterByAnonymizationLevel(spotlightData) {
+	const firstFiveRecommendedLoans = spotlightData.lend?.loanChannelsById[0]?.loans?.values ?? [];
+	const nonAnonymousLoans = firstFiveRecommendedLoans.filter(
+		loan => loan.anonymizationLevel !== 'full'
+	);
+	return nonAnonymousLoans[0] || {};
+}
+
+export default {
+	name: 'LoanSpotlight',
+	props: {
+		categorySlug: {
+			type: String,
+			default: ''
+		},
+		fallbackCategorySlug: {
+			type: String,
+			default: ''
+		}
+	},
+	components: {
+		KvButton,
+		KvResponsiveImage,
+		KvLoadingPlaceholder,
+		KvLoadingParagraph
+	},
+	inject: ['apollo', 'cookieStore'],
+	data() {
+		return {
+			spotlightPlaceholderImageCTF: '',
+			spotlightLoan: {},
+			allChannelsData: {},
+			isLoading: true,
+			targetedLoanChannelID: null
+		};
+	},
+	computed: {
+		altText() {
+			return "Today's loan spotlight";
+		},
+		getSpotlightLoanID() {
+			return this.spotlightLoan.id ?? '';
+		},
+		getSpotlightText() {
+			return toParagraphs(this.spotlightLoan.description) ?? '';
+		},
+		locationExists() {
+			return this.spotlightLoan.geocode ?? '';
+		},
+		getSpotlightLoanLocation() {
+			if (this.locationExists !== '') {
+				if (this.spotlightLoan.geocode.city && this.spotlightLoan.geocode.country.name) {
+					return `${this.spotlightLoan.geocode.city}, ${this.spotlightLoan.geocode.country.name}`;
+				}
+				return `${this.spotlightLoan.geocode.country.name ?? ''}`;
+			}
+			return '';
+		},
+		imageExists() {
+			return this.spotlightLoan.image?.default ?? '';
+		},
+		getSpotlightImage() {
+			if (this.imageExists === '') {
+				return [['small', this.spotlightPlaceholderImageCTF]];
+			}
+			return [['small', this.spotlightLoan.image.retina], ['small retina', this.spotlightLoan.image.retina]];
+		}
+	},
+	apollo: {
+		query: allChannelsQuery,
+		preFetch: true,
+		result(result) {
+			this.allChannelsData = result.data?.lend?.loanChannels?.values ?? [];
+		},
+	},
+	created() {
+		// eslint-disable-next-line max-len
+		this.targetedLoanChannelID = getTargetedChannel(this.categorySlug, this.fallbackCategorySlug, this.allChannelsData);
+
+		this.apollo.query({
+			query: spotlightLoanQuery,
+			variables: {
+				ids: [this.targetedLoanChannelID],
+			},
+		}).then(result => {
+			// filter out loans with anonymizationLevel of full, then take first in list
+			this.isLoading = false;
+			this.spotlightLoan = filterByAnonymizationLevel(result.data);
+		});
+	},
+	mounted() {
+		this.apollo.query({
```

Comment:
I think you can wrap this query in an if getSpotlightImage exists -- since created will have run on the server and you don't need to do this query if an image exists

Reply:
I thought about this a little more --  Lets remove all the placeholder image code. Let's filter to only display loans that have an image. If a loan doesnt have an image it's not spotlight worth, lets not show it

Reply:
I agree! I was thinking the same thing.